.github/workflows/unit-integration-tests.yml

```diff
@@ -20,6 +20,7 @@ env:
   # without having to download a newer one.
   GOTOOLCHAIN: local
   GODEBUG: "x509negativeserial=1"
+  TEST_RESULT_PATH: /tmp/test-results
```

Comment:
With some added code, we're using the value `/tmp/test-results` in a few more places than we were before. Moving this to the outermost env allows us to reuse it as a variable.